target/linux/bcm63xx/base-files/etc/board.d/02_network

```diff
@@ -15,7 +15,8 @@ netgear,cvg834g|\
 netgear,dgnd3700-v2|\
 netgear,evg2000|\
 t-com,speedport-w-303v|\
-t-com,speedport-w-500v)
+t-com,speedport-w-500v|\
+technicolor,tg582n)
```

Comment:
Please, add a proper switch configuration (with VLANs).
If the none of the 4 ethernet ports is labelled as WAN, then the 4 ethernet ports should be added to eth0.1.